<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_Setup">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_persp_setup_unc.png</normaloff>
       <activeon>icons/icon_persp_setup_clc.png</activeon>
       <selectedon>icons/icon_persp_setup_clc.png</selectedon>icons/icon_persp_setup_unc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_Segmentation">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_persp_segm_unc.png</normaloff>
       <activeon>icons/icon_persp_segm_clc.png</activeon>
       <selectedon>icons/icon_persp_segm_clc.png</selectedon>icons/icon_persp_segm_unc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_Annotation">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_persp_ann_unc.png</normaloff>
       <activeon>icons/icon_persp_ann_clc.png</activeon>
       <selectedon>icons/icon_persp_ann_clc.png</selectedon>icons/icon_persp_ann_unc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_Classification">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_persp_class_unc.png</normaloff>
       <activeon>icons/icon_persp_class_clc.png</activeon>
       <selectedon>icons/icon_persp_class_clc.png</selectedon>icons/icon_persp_class_unc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_Query">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_persp_query_unc.png</normaloff>
       <activeon>icons/icon_persp_query_clc.png</activeon>
       <selectedon>icons/icon_persp_query_clc.png</selectedon>icons/icon_persp_query_unc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_WebApp">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_persp_webapp_unc.png</normaloff>
       <activeoff>icons/icon_persp_webapp_clc.png</activeoff>icons/icon_persp_webapp_unc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
